samples/tools/testbed/utils/metrics_human_eval.py

```diff
@@ -0,0 +1,98 @@
+import os
+import sys
+import argparse
+import csv
+
+
+def metrics(results_fh):
+    """
+    Compute metrics from collated HumanEval results.
+
+    Args:
+        results_fh (File Stream): A file stream containing the collated results in CSV.
+    """
+
+    reader = csv.reader(results_fh)
```

Comment:
Will be good to give the format for the CSV file. The code below is hard to read.

Reply:
Good idea. Will add.